src/gpio.rs

```diff
@@ -1,4 +1,39 @@
 //! GPIO and pin configuration
+//!
+//! Interface for the input/output pins.
+//!
+//! `Gpio1` through `Gpio39` represent the *physical* pins of the ESP chip.
```

Comment:
Also maybe "... represent the *physical* pins or in other words - the pins *peripherals* of the ESP chip".

Reply:
I never said to remove the physical notion, but to add to it that we mean a peripheral, which is the notion we use throughout the hal.